.vscode/settings.json

```diff
@@ -4,5 +4,8 @@
   "stylelint.validate": ["css", "less", "postcss", "vue"],
   "tailwindCSS.experimental.classRegex": [
     ["cva\\(([^)]*)\\)", "[\"'`]([^\"'`]*).*?[\"'`]"]
-  ]
+  ],
+  "[vue]": {
```

Comment:
Why did you need to explicitly set it again for `vue` files? It should be the default formatter for all files already ...

Reply:
It didn't work for me 😅

Reply:
Well, neither this seems to work on your machine... The linter fails for a couple of files of several extensions, a.o. `vue`.

Reply:
Hm ok, I'll take a look